recipes/graphite_plugin.rb

```diff
@@ -48,6 +48,5 @@
     -c #{node['bareos']['plugins']['graphite']['config_path']}/graphite-poller.conf
   EOH
   mailto node['bareos']['plugins']['graphite']['mail_to']
-  user 'bareos'
-  minute '5'
```

Comment:
I figured it didn't need to run as bareos as long as the mailto was set to go to whatever alias is being pointed to via attributes. Also, I want the cronjob to run minutely instead..